### PR TITLE
Add an icon to admin menu.

### PR DESCRIPTION
**Pull request by jlamur**
This is my first PR, I'm sorry if I do something in the wrong way !

I am targetting this branch, because it's a minor change that does not break BC.

## Changelog

```markdown
### Added
- An icon to admin menu (fa-image).
```

## Subject

There was no image in the admin menu. So I added one :)

### CODE REVIEW

File: Resources/config/doctrine_mongodb_admin.xml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
     <parameters>
+        <!-- COMMON -->
+        <parameter key="sonata.media.admin.groupname">sonata_media</parameter>
```
Comment (greg0ire) [inline Resources/config/doctrine_mongodb_admin.xml]: What does this have to do with the issue at hand exactly?

Comment (greg0ire) [inline Resources/config/doctrine_mongodb_admin.xml]: I don't know what interest it has exactly, but if you find one, then make sure it appears in your commit message. As a rule of thumb, try to have commit messages that look like this : 

```
What you are doing
<blank line>
Why you are doing it
```
More information about commit messages can be found in the CONTRIBUTING.md

Anyway, nice first PR, it is quite clean (cleaner than the average PR IMO).

**Comment by OskarStark** [comment]
how do we set the icon for the other bundles? is it done in this way?

**Comment by jlamur** [comment]
@OskarStark Yes. It's almost a copy/paste from SonataUserBundle. 

It is also the reason why I introduced the parameter `sonata.media.admin.groupname` (see @gregoire comment)... There's a parameter `sonata.user.admin.groupname` in the user bundle.

**Comment by greg0ire** [comment]
> It is also the reason why I introduced the parameter sonata.media.admin.groupname (see @gregoire comment)... There's a parameter sonata.user.admin.groupname in the user bundle.

Then your commit message should read

```
Add parameter for group name

Let's be consistent with the user bundle.
```

**Comment by jlamur** [comment]
Thank you for theses advices @greg0ire.
I'll try to follow your recommendations, then edit my commits and this pull request when I'll have the time to.

**Comment by greg0ire** [comment]
You're welcome. The way I see it, the easiest way for you to do that would be : 

1. forget the commits (but not the changes) : `git reset reference to the last commit that isn't yours`
2. prepare commits one by one : `git add -p`, then `git commit`
3. overwrite anything that is on this github branch with your new work : `git push --force`

**Comment by jlamur** [comment]
Done :).

**Comment by jordisala1991** [comment]
Is this PR waiting for something or we can merge it?

**Comment by greg0ire** [comment]
@sonata-project/contributors please review

**Comment by greg0ire** [comment]
Thanks @jlamur !
